app/views/media/show.html.haml

```diff
@@ -19,7 +19,7 @@
 
         %p= t('.license_context', link: link_to(t('.license_context_link'), 'https://creativecommons.org/licenses/by/3.0/')).html_safe
 
-        = link_to(t('.download_jpeg').capitalize + ' ' + t('.download_res_original'), "#{ENV['IMAGE_SERVICE_URL']}/#{@image.graph_id}.jpeg?download=true", class: 'btn--primary')
+        = link_to(t('.download_jpeg').capitalize + ' ' + t('.title') + ' ' + @person.display_name + ' ' + t('.download_format_original') + ' ' + t('.download_res_original'), "#{ENV['IMAGE_SERVICE_URL']}/#{@image.graph_id}.jpeg?download=true", class: 'btn--primary')
```

Comment:
The link text have become too long, I think we should preserve the shorter link text for visually-abled user, and have a better description text within an aria-label for assistive technology users.